### PR TITLE
Bump slo-monitor to 0.15.7

### PR DESCRIPTION
**Pull request by CoderSherlock**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

To fix CVE-2023-45285 and CVE-2023-39326

#### Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: golang/Makefile [hunk 1]
```diff
@@ -15,7 +15,7 @@
 .PHONY: build-go build push
 
 export GCS_BUCKET?=k8s-infra-scale-golang-builds
-export GO_COMPILER_PKG?=go1.21.3.linux-amd64.tar.gz
+export GO_COMPILER_PKG?=go1.21.5.linux-amd64.tar.gz
```
Comment (ruiwen-zhao) [inline golang/Makefile:18]: is this used by slo monitor?

File: slo-monitor/Makefile [hunk 2]
```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 PACKAGE = k8s.io/perf-tests/slo-monitor
-TAG = 0.15.6
+TAG = 0.15.7
```
Comment (ruiwen-zhao) [inline slo-monitor/Makefile:16]: (Can't comment on files you didn't change) Can you update the go.mod file under slo-monitor from `go 1.20` to 1.21? IIUC when building, Louhi will automatically use the latest patch version of go 1.21.

Comment (CoderSherlock) [reply slo-monitor/Makefile:16]: I just checked the Louhi execution logs on previous build. It still build using 1.21.3 with specifying `1.20` in go.mod. I assume, we just need to update the tag version to trigger another build. wdyt?

Comment (ruiwen-zhao) [reply slo-monitor/Makefile:16]: yep the version in go.mod indicates the minimum go version required. Yeah, let's just update the tag and re-trigger louhi build.

Comment (CoderSherlock) [reply slo-monitor/Makefile:16]: Removed irrelevant code. PTAL.

**Review by ruiwen-zhao — APPROVED** [review]

**Comment by tosi3k** [comment]
/lgtm
/hold

Could you update the version included in the PR title to 0.15.7?

**Comment by CoderSherlock** [comment]
Updated the commit message and PR title.

**Pull request merged**